features/vaadin-surveillance-views/src/main/java/org/opennms/features/vaadin/surveillanceviews/config/SurveillanceViewProvider.java

```diff
@@ -0,0 +1,207 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2014 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+package org.opennms.features.vaadin.surveillanceviews.config;
+
+import org.opennms.features.vaadin.surveillanceviews.model.SurveillanceViewConfiguration;
+import org.opennms.features.vaadin.surveillanceviews.model.View;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.xml.bind.JAXB;
+import java.io.File;
+
+/**
+ * This class is used for loading, holding and saving of {@link SurveillanceViewConfiguration} definitions.
+ *
+ * @author Christian Pape
+ */
+public class SurveillanceViewProvider {
+    /**
+     * the logger instance
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(SurveillanceViewProvider.class);
+    /**
+     * Instance variable for this singleton object
+     */
+    private static SurveillanceViewProvider m_surveillanceViewProvider = new SurveillanceViewProvider();
+    /**
+     * A variable fro holding a {@link SurveillanceViewConfiguration} instance
+     */
+    private SurveillanceViewConfiguration m_surveillanceViewConfiguration = null;
+    /**
+     * The configuration {@link java.io.File} to be used.
+     */
+    private File m_cfgFile = new File("etc/surveillance-views.xml");
```

Comment:
In my opinion we not need the dao anymore because all the required code for the new 
surveillance-view/dashboard is inside this feature including the jaxb-stuff. Will this break 
something?